docs/content/token/staking/index.mdx

```diff
@@ -16,7 +16,94 @@ it to register as a node operator or delegator.
+    * Tokens Staked: The tokens that a user has staked in the current epoch.
+    * Tokens Requested to Unstake: The amount of tokens that a user has requested to be unstaked
+      at the end of the current epoch.
+    * Tokens Unstaking: The tokens that were unstaked at the beginning of the current epoch and
+      are being held for an additional epoch holding period before being released.
+    * Tokens Unstaked: Tokens that used to be committed or staked and have been unstaked.
+    * Tokens Rewarded: Tokens that the user has received via staking rewards.
+
+**Epoch:** The period of time between reward payments and changes in the identity table. (Initially a week)
+At the end of every epoch, insufficiently staked node operators are refunded their stake,
+rewards are paid to those who are currently staked, committed tokens are marked as staked, 
+unstaking tokens are marked as unstaked, and unstaking requests are changed from staked to unstaking.
+
+**Delegation Rewards Cut:** The percentage of a delegator's rewards that the node operators take. Initially set to 10%.
+
+**Epoch Payout:** The total amount of tokens paid in rewards at the end of an epoch. 
+This value will change as the supply of FLOW changes.
+
+**Minimum Stake Requirement:** Each node type has a requirement for the minimum number of FLOW
+they have to commit to stake to be considered a valid node and receive rewards. If a node operator
+does not meet the minimum stake, they will not be included in the next epoch and will not receive any rewards.
+Delegators are not subject to minimum stake requirements.
+
+- Collector Nodes: 250,000 FLOW
+- Consensus Nodes: 500,000 FLOW
+- Execution Nodes: 1,250,000 FLOW
+- Verification Nodes: 135,000 FLOW
+
+### Smart Contract Summary
+
+The Flow staking smart contract manages a record of stakers who have staked tokens for the network. 
+Users who want to stake can register with the staking contract at any time, and their tokens will
+be locked for staking until they request to unstake them.
+
+Before getting into the details of the contract, we need to discuss how time in Flow is organized.
+The operation of the Flow network is divided into epochs, which are week-long periods where various
+network maintenance operatios are performed. The schedule of each epoch is exactly the same, though
+individual actions that users take in relation to the network will be different.
+
+Epoch Schedule:
+
+1. Start of Epoch: Generic metadata about the current epoch is updated and shared.
+2. Staking Auction: Stakers can perform any action they want to manage their stake, like
+    initially registering, staking new tokens, unstaking tokens, or withdrawing rewards.
+    This phase takes up the vast majority of time in the epoch.
+3. Remove Insufficiently Staked Nodes: All node operators who don't meet the minimum will be removed.
+4. Rewards Payout: Pay rewards to all the node operators staked in the current epoch.
```

Comment:
Just for completeness it might be worth putting where in this workflow the node is actually part of the current epoch and servicing requests.